src/lib.rs

```diff
@@ -685,6 +684,34 @@ impl Uuid {
         Uuid { bytes: b }
     }
 
+    /// Creates a v4 Uuid from rng bytes (ref issue #170)
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// use uuid::Uuid;
```

Comment:
I still think we should add an example that's a little more motivating for people using this function, I don't think the value of the bytes we pass in, nor the assertion, should be visible in the remdered docs.

Reply:
I just copied it from the examples given for the other functions. Will change it accordingly.